docs/duration.md

```diff
@@ -316,6 +337,36 @@ function yearsToMonths(duration) {
 yearsToMonths(threeYears).minus(yearsToMonths(oneAndAHalfYear))  // => P18M
 ```
 
+### duration.**negated**() : Temporal.Duration
+
+**Returns:** a new `Temporal.Duration` object with the opposite sign.
+
+This method gives the negation of `duration`.
+It returns a newly constructed `Temporal.Duration` with all the fields having the opposite sign (positive if negative, and vice versa.)
+If `duration` is zero, then the returned object is a copy of `duration`.
+
+Usage example:
+```javascript
+d = Temporal.Duration.from('P1Y2M3DT4H5M6.987654321S');
+d.sign  // 1
+d.negated()  // -P1Y2M3DT4H5M6.987654321S
+d.negated().sign  // -1
+```
+
+### duration.**abs**() : Temporal.Duration
+
+**Returns:** a new `Temporal.Duration` object that is always positive.
+
+This method gives the absolute value of `duration`.
+It returns a newly constructed `Temporal.Duration` with all the fields having the same magnitude as those of `duration`, but positive.
+If `duration` is already positive or zero, then the returned object is a copy of `duration`.
+
+Usage example:
+```javascript
+d = Temporal.Duration.from('-PT8H30M');
+d.abs()  // PT8H30M
+```
+
 ### duration.**getFields**() : { years: number, months: number, weeks: number, days: number, hours: number, minutes: number, seconds: number, milliseconds: number, microseconds: number, nanoseconds: number }
 
 **Returns:** a plain object with properties equal to the fields of `duration`.
```

Comment:
I can't leave a review comment on `toString` so leaving it here as it's the closest place. Anyway, `toString` docs should probably be expanded to discuss negative durations. Here's some possible content: 

* Positive durations are ISO 8601 compliant, with no leading plus sign. Example: `P1DT12H`.
* Negative durations will be serialized with a leading minus sign. This is an extension to ISO 8601 that is accepted by most other date/time libraries in both JavaScript and other platforms. Example: `-P1DT12H`.
* Zero durations will be serialized as `PT0S`.